### ISSUE TITLE
Wiki Sidebar Update

### ISSUE THREAD

**Issue by MeepyMay** [systers/language-translation#268]
## Description
As a newcomer,
I need to navigate the repo successfully,
so that I can understand where the project is in its development.

## Mocks
![screen shot 2018-04-19 at 3 47 20 pm](https://user-images.githubusercontent.com/2440938/39014663-06dcb230-43e9-11e8-997c-96a89b7c55e8.png)

### Update [Required]
- [ ] Update to [the template](https://docs.google.com/document/d/1h53P1h5g9hWEUsnOhpl9PgVzbWEGhTMIP2Pyta-MYSA/edit?usp=sharing) as discussed by the Community.
### Enhancement to Update [Optional]
- [ ] Link all GSoC/Outreachy student's pages if applicable.
- [ ] Link Final Reports page if applicable.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 3 Community Members.

## Estimation
10-30 minutes

**Comment by afreen23** [comment]
Hey, I am working on this issue

**Comment by Fenn-CS** [comment]
@afreen23 yeah go ahead... No one has claimed the issue. @ExactlyMay ps you want to change the status of this issue.

**Comment by afreen23** [comment]
How would I add sidebar until I get access to the repository? Should I start with markdown and then the sidebar will be added ?

**Comment by afreen23** [comment]
Okay I am working on this !
